_maps/map_files/RandomRuins/SpaceRuins/abandonedtele.dmm

```diff
@@ -0,0 +1,311 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/template_noop,
+/area/template_noop)
+"b" = (
+/turf/simulated/wall/r_wall,
+/area/AIsattele)
```

Comment:
The spaceruin should not use the same /area as the AI core on station.
It should use /area/ruin/tcommsat. And that area should be named "Abandoned Telecomms Sat" or similar to stop people who appear there on crew monitors, GPS, etc from being thought to be on the AI sat.

Reply:
This is the area the ruin used already, but I can change it if it’s needed.